### PR TITLE
(#106) Added a student project section and fixed header spacing

### PR DESCRIPTION
**Pull request by AppleC4T**
- [x] Included a Preview link and screenshot showning after and before the changes.
- [x] Included a description of change below.
- [x] Squashed the commits.

# Changes done in this Pull Request

- If your change will be reflected on the website, please provide a **Test-Link** (**Hint : `gh-pages`**)
- Fixes https://github.com/fossasia/gci18.fossasia.org/issues/106
- Preview Link: https://applec4t.github.io/ACgci18fossasia/

Before:
image
After:
image

## Description / Changes
Fixed sections headers spacings by adding a few `<br>`  tags. Added a student projects section using jekyll, added _data file for adding new projects for display, added img folder for project icons. The current entries in the section are blank, test entries for showing the scheme and can be (or even should be) replaced by actual projects later.

### CODE REVIEW

File: _data/student-projects.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+#   desc: a short description of the project
+#   github: link to student's project github repo (copy the whole link)
+#   link: link to student's project website
+
+- name: gci18.fossasia.org
+  author: gci18 fossasia students
+  desc: A website made for the Google Code-In 2018 contest. Contributed to both by fossasia students and mentors.
+  link: gci18.fossasia.org
+  github: https://github.com/fossasia/gci18.fossasia.org
+
```
Comment (STEdirisuriya) [inline _data/student-projects.yml]: No need of an additional space.

File: index.html [hunk 2]
```diff
@@ -159,6 +164,62 @@ <h2> Students </h2>
...
+                            <img src="img/studentsprojects/{{sproj.image}}" alt="{{ sproj.name }}">
+                            {% else %}
+                            <img src="img/studentprojects/spdeafult.png" alt="{{ sproj.name }}">
+                            {% endif %}
+                            <br>
+                            <p class="person-name">{{ sproj.name }}</p><br>
+							{% if sproj.author %}
+                                <p>by<strong>{{ sproj.author }}</strong></p>
+                                {% endif %}
+							<br>
```
Comment (STEdirisuriya) [inline index.html]: Indent the html file properly.

**Comment by AppleC4T** [comment]
@STEdirisuriya done. I changed the indentation and hovering on the project image no longer changes the display. 👍

File: _data/student-projects.yml (hunk 1 above)
Comment (STEdirisuriya) [reply _data/student-projects.yml]: See this

**Comment by STEdirisuriya** [comment]
@AppleC4T There is a my another comment here. Please look it. "Remove additional spaces"

**Comment by STEdirisuriya** [comment]
I am not sure others like this, I think the best way to show projects is adding those as row placeholders rather than one by one down. Please wait until others opinions. Other things good to me.

**Comment by niteshkumarniranjan** [comment]
@jogendra I have already created it but someone's PR has removed the link. Check it out: https://gci18.fossasia.org/projects.html

**Comment by AppleC4T** [comment]
@niteshkumarniranjan so maybe what we could do is I’ll change my PR to make a short section linking to your student projects page and change the Jekyll script so you can add descriptions to projects. What do you think?

**Comment by jogendra** [comment]
@niteshkumarniranjan Keep the projects in the same page. Do not create a new page for that. Some already made PR for that I think.

**Comment by AppleC4T** [comment]
@jogendra should I keep the section and just change the alignment then?

**Comment by jogendra** [comment]
@AppleC4T Yes!

**Comment by AppleC4T** [comment]
@jogendra done. The project cards now stack vertically and I resolved the merge conflicts.

**Comment by AppleC4T** [comment]
@jogendra ok, I'll make a new PR in a moment, though, because my fork got outdated and I get a ton of conflicts while squashing commits.

**Pull request closed without merging**